Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN apt-get update && apt-get install -y \
 RUN pip install --no-cache-dir poetry==1.1.13
 COPY . /app
 WORKDIR /app
-# For now while we are in heavy development we install the latest with Poetry 
+# For now while we are in heavy development we install the latest with Poetry
 # and execute directly with Poetry. Later, we'll move to the released Pip package.
-RUN poetry install -E preql -E mysql -E pgsql -E snowflake
+RUN poetry install
```

Comment:
This installs all dev dependencies anyway; not necessary to do the extras anymore.